### ISSUE TITLE
Black screen

### ISSUE THREAD

**Issue by JC3000L** [SMEDjs/ow-vision#3]
I'm not sure if I'm just being stupid or what but when I load everything up, it works for a solid 4-5 seconds and then the AI view goes completely black. Not sure what is causing it. 
https://prnt.sc/WeNkxQfp9ThQ

EDIT: The black screen happens when I load into the practice range but does not stop when I leave the practice range. Game needs to be restarted for it to fix. Then blacks out again when I load in.

Also if you don't mind me asking, where do I use the !yolo training command? Pretty new to this stuff.

**Comment by SMEDjs** [comment]
Yes, this is caused by Overwatch anticheat it's called screenlock and I don't know how to bypass it, I never got it but some users did 🤷

**Comment by krytoi945** [comment]
where to use !yolo        ???

**Comment by G59Dev** [comment]
same here black screen windows 11 on steam

**Comment by G59Dev** [comment]
fixed it jusst use full screen exclusive
